include/variables.h

```diff
@@ -133,19 +133,19 @@ extern UNK_PTR D_801BEC14;
 // extern UNK_TYPE1 D_801BED00;
 // extern UNK_TYPE2 D_801BED24;
 // extern UNK_TYPE1 D_801BED3C;
-// extern UNK_TYPE1 D_801BED40;
-// extern UNK_TYPE1 D_801BED4C;
+// extern UNK_TYPE1 mapColorTable;
+// extern UNK_TYPE1 gameplay_dangeon_keep_Info;
```

Comment:
Could you rename those symbols to follow the naming style?

Reply:
Also, the ones that are meant to be global should be moved to the map header, the others should be removed from here